### DISCUSSION TITLE
How to turn off checking python imports?

### DISCUSSION THREAD

**Discussion by igor-yusupov** [zed-industries/zed#6529] [Feature Requests]
How to turn off checking python imports?

**Comment by hovsater** [comment]
Zed uses [pyright](https://microsoft.github.io/pyright) as the language server for Python. pyright can be configured using a `pyrightconfig.json` file in the root of your project. You can find the available configuration options that go in that file [here](https://microsoft.github.io/pyright/#/configuration?id=main-configuration-options).

I believe what you're looking for is the configuration option `reportMissingImports`. It's default value is `true` but can easily be set to `false`. See this [sample configuration](https://microsoft.github.io/pyright/#/configuration?id=sample-config-file) for inspiration.

**Comment by igor-yusupov** [comment]
@hovsater it works, thank you!
